http/servlet-undertow/src/test/java/io/quarkus/ts/http/undertow/HttpServletWithSessionListenerIT.java

```diff
@@ -51,6 +57,30 @@ public void sessionSecured() {
         thenCheckActiveSessionsEqualTo(1);
     }
 
+    @Test
+    @DisplayName("undertow servlet test http2")
+    public void underTowHttp2Servert(VertxTestContext vertxTestContext) {
+        final String HTTP_2_VERSION = "HTTP_2";
+        Vertx vertx = Vertx.vertx();
+        String uriUnderTow = "/app/servlet/hello";
+        String expectedResponseFromUndertow = "Hello World";
+        HttpClientOptions options = new HttpClientOptions().setProtocolVersion(HttpVersion.HTTP_2);
+        HttpClient httpClient = vertx.createHttpClient(options);
+        httpClient.request(HttpMethod.GET, 1101, "localhost", uriUnderTow)
+                .compose(request -> request.send()
+                        .compose(httpClientResponse -> {
+                            System.out.println("Got response {} with protocol {}" + httpClientResponse.statusCode() + " "
```

Comment:
We are using logger, I was never actually sure why, but if you dig in (or ask around) I'm quite sure there is a good reason for it (not just features!)